Dockerfile

```
@@ -0,0 +1,3 @@
+FROM gcr.io/distroless/static-debian12:nonroot
+COPY task /task
```

I don't know if the build is "hardened", let's just make sure that users can read and execute automatically
```suggestion
COPY --chmod=555 task /task
```

---

Since its based on distroless,  would you ever run it by itself though? I think this would maybe make sense with a usable base. Dunno